music21/meter/core.py

```diff
@@ -419,10 +419,6 @@ def __init__(self, value=None, partitionRequest=None):
     # SPECIAL METHODS #
 
     def __eq__(self, other) -> bool:
-
-        if not super().__eq__(other):
-            return False
-
         return True
```

Comment:
I still don't understand this -- what does this help in the system?  I don't know of any object that always compares True to every other object.

Reply:
Hi @mscuthbert .

Thanks for this. As I say, this odd little method is just a placeholder - not the final solution, but a demo/reminder that something is needed on this class and none of the following are acceptable alternatives:
- `def __eq__` with a simple `pass` 
- no `def __eq__` but with `equalityAttributes = ()` on the class instead
- neither of those two.

Excuse the exploratory approach but I don't know why this behaves as it does - whether it's to do the new wider handling of eq you've started implementing, or something on meter.

In short, I'm just reporting on the behaviour I see for this case and what 'works' (i.e., what apparently is needed to get the musical tests returning the correct bools).

My hope is that having this oddity and discussion point helps us diagnose what's going on and replace it with something more plausible. Likely it will feed into your wider re-working of meter.

The rest seems to work well ... and without oddities.

Thanks for any ideas for a fix. Equally, just say if you want to drop this PR, e.g., if it's not a good time for it / doesn't fit with your root-and-branch reform of the meter trees ... pun intended ;)

Reply:
I think it's best to comment it out.  I've never heard of an object where `if OBJ == True` is the same as `if OBJ == False` and `if OBJ == 5` and `if OBJ == 2` are all True.  It leads to huge issues with using the obj in a hash situation.